pyccl/baryons/vandaalen19.py

```diff
@@ -0,0 +1,94 @@
+__all__ = ("vanDaalen19",)
+
+import numpy as np
+
+from .. import Pk2D
+from . import Baryons
+
+
+class vanDaalen19(Baryons):
+    """The "van Daalen+ 2019" model boost factor for baryons.
+
+    .. note:: First presented in van Daalen et al., arXiv:1906.00968.
```

Comment:
Can we include a link? See e.g. https://github.com/LSSTDESC/CCL/blob/dd2aa5e0cc228505f85daa8d869f8706bd4d8e37/pyccl/baryons/schneider15.py#L13

Reply:
Done and recompiled docs